src/Pages/ChooseClass.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import "../Stylesheets/App.css";
+import Tile from "../Components/Tile.js";
+
+import idiot from "../Images/idiot.png";
+import biggestIdiot from "../Images/biggestIdiot.png";
+import ericWilson from "../Images/ericWilson.png";
+
+import { Analytics } from "aws-amplify";
+
+class ChooseClass extends Component {
+  render() {
+    Analytics.record("appRender");
+    return (
+      <div className="content">
+        <h1 className="pageHeader">Pick a class</h1>
+        <ul className="tiles-section">
+          <Tile
```

Comment:
You could DRY this up by mapping a pre-defined array of objects (even imported from a neighboring file. It'd be something like this (syntax not guaranteed): 
```jsx
const classProps = [
  {
    image: idiot,
    link: '/chooseAttributes',
    text: 'Barbarian',
  },
  {
    image: ericWilson,
    link: '/chooseAttributes',
    text: 'Bard',
  },
  ...etc,
];

<InDatComponentWhatever>
  {classProps.map((demProps, i) => (<Tile id={i} {...demProps} />))}
</InDatComponentWhatever>
```